src/ciphers/caesar.rs

```diff
@@ -1,43 +1,115 @@
-//! Caesar Cipher
-//! Based on cipher_crypt::caesar
-//!
-//! # Algorithm
-//!
-//! Rotate each ascii character by shift. The most basic example is ROT 13, which rotates 'a' to
-//! 'n'. This implementation does not rotate unicode characters.
-
-/// Caesar cipher to rotate cipher text by shift and return an owned String.
-pub fn caesar(cipher: &str, shift: u8) -> String {
-    cipher
+const ERROR_MESSAGE: &str = "Rotation must be in the range [0, 25]";
+const ALPHABET_LENGTH: u8 = b'z' - b'a' + 1;
```

Comment:
Do you know some _reasonable_ way to check if `ALPHABET_LENGTH` is indeed `26`?

Reply:
Only direct comparison in the code

Reply:
This has to be placed inside a test. What do you think about simply adding a suitable test with
```rust
assert_eq!(ALPHABET_LENGTH, 26);
```